apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/offerbook/MarketSortType.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.main.content.bisq_easy.offerbook;
+
+import lombok.Getter;
+
+import java.util.Comparator;
+
+@Getter
+enum MarketSortType {
```

Comment:
What do you think about putting all the enums together in one single file? That was my thinking with the `Filters` class, to be a wrapper of all the enums. We could rename it to `FiltersAndSorts` and include this one